cypress/support/fragments/invoices/newBatchGroup.js

```diff
@@ -0,0 +1,8 @@
+import getRandomPostfix from '../../utils/stringTools';
+
+export default {
+  defaultUiBatchGroup : {
+    name: `000autotest_group_${getRandomPostfix()}`,
```

Comment:
000 related with searching/filtering? Do we really need this prefix?

Reply:
Yes, we need it for quick sorting.

Reply:
if you can filter values by name - it's more convenient then  keeping of special names with sorting support

Reply:
There is no filtering option in this page. We can just search for some cell with text in verification.